### PR TITLE
Add user's sessions and login form

### PR DESCRIPTION
**Pull request by trungph-0971**
user_sessions

### CODE REVIEW

**Comment by trungph-0971** [comment]
ready

File: app/views/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for :session, url: login_path do |f| %>
+
+      <%= f.label t(".email") %>
+      <%= f.email_field :email, class: 'form-control' %>
+
+      <%= f.label t(".password") %>
+      <%= f.password_field :password, class: 'form-control' %>
```
Comment (phongnvk-0916) [inline app/views/sessions/new.html.erb]: ' => "

File: app/views/sessions/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<% provide :title, t(".title") %>
+<h1><%= t ".h1" %></h1>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for :session, url: login_path do |f| %>
+
+      <%= f.label t(".email") %>
+      <%= f.email_field :email, class: 'form-control' %>
```
Comment (phongnvk-0916) [inline app/views/sessions/new.html.erb]: ' => "

**Comment by trungph-0971** [comment]
ready

**Pull request merged**